spec/must_support_spec.rb

```diff
@@ -1,5 +1,6 @@
 RSpec.describe CancerPathologyDataSharingTestKit::MustSupportTest do # rubocop:disable RSpec/SpecFilePathFormat
```

Comment:
```suggestion
RSpec.describe CancerPathologyDataSharingTestKit::MustSupportTest, :runnable do # rubocop:disable RSpec/SpecFilePathFormat
```

Do this rather than explicitly including the context.